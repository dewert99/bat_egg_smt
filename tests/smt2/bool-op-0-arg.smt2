(define-const xor* Bool xor)
(define-const and* Bool and)
(define-const or* Bool or)
(check-sat)
(get-model)