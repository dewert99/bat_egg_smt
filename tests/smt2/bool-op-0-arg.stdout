sat
(
 (define-fun and* () Bool true)
 (define-fun or* () Bool false)
 (define-fun xor* () Bool false)
)
